src/main/java/seedu/address/model/person/ClassTiming.java

```diff
@@ -156,4 +216,23 @@ public int hashCode() {
         return value.hashCode();
     }
 
+
+    @Override
+    public int compareTo(ClassTiming o) {
+        int thisDayInt = getDayToInt();
+
+        String otherDay = o.getDay();
+        int otherDayInt = o.getDayToInt();
```

Comment:
I think it's better to store Day as a class enum which has both attribute `String day` and `int dayInt`